src/main/java/net/imglib2/algorithm/morphology/table2d/Spur.java

```diff
@@ -73,7 +73,7 @@
 
 		final ImgFactory< T > factory;
 		if ( extendedVal instanceof NativeType )
-			factory = ( ImgFactory< T > ) Util.getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
+			factory = Util.< NativeType >getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
```

Comment:
Insane! But I can easily top that with Eclipse:
If you have
```
( ImgFactory< T > ) Util.getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
```
and tell Eclipse to "remove unnecessary casts", it will remove the "unnecessary cast".
```
Util.getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
```
then complain with a compile error...